### PR TITLE
Add keyboard functionality to dev-site search

### PR DESCRIPTION
**Pull request by dev-cprice**
### Summary

While yes, https://github.com/cerner/terra-dev-site/issues/178 has already been closed, the site search still felt like it needed keyboard a11y... so I added it real quick. 😀

[CONTRIBUTORS.md]: https://github.com/cerner/terra-dev-site/blob/master/CONTRIBUTORS.md
[License]: https://github.com/cerner/terra-dev-site/blob/master/License.md

### CODE REVIEW

File: src/search/_Search.jsx [hunk 1]
```diff
@@ -24,6 +24,12 @@ const propTypes = {
   onItemSelected: PropTypes.func.isRequired,
 };
 
+const ENTER = 13;
```
Comment (emilyrohrbough) [inline src/search/_Search.jsx]: We should use the keycodes pacakge we use in other terra components.

**Comment by dev-cprice** [comment]
@emilyrohrbough for sure, meant to leave a comment on this PR stating that tests were coming last night 👍

File: src/search/_Search.jsx (hunk 1 above)
Comment (dev-cprice) [reply src/search/_Search.jsx]: Seeing as we already include it in the [dev-site bundle](https://github.com/cerner/terra-dev-site/blob/master/package.json#L100), I'll go ahead and change this

didn't use initially when prototyping this out

**Comment by emilyrohrbough** [comment]
@dev-cprice Do you have an update for this PR?

**Comment by dev-cprice** [comment]
@emilyrohrbough not at the moment, been tied up with quarterly commits

hoping to get back to this soon-ish in my free time

**Comment by emilyrohrbough** [comment]
@dev-cprice  are you okay with close the review until it's ready for review?

**Comment by dev-cprice** [comment]
@emilyrohrbough For sure, that's fine.

**Comment by emilyrohrbough** [comment]
Closing PR for now till @dev-cprice has time to pick up the tests for this change.

**Pull request closed without merging**